ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioStaging.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1420"
+   LastUpgradeVersion = "1520"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `LastUpgradeVersion` has been updated. This is usually done when the Xcode version used to build the project is updated. Ensure that this change is intentional and all team members are using the compatible Xcode version.
```diff
-   LastUpgradeVersion = "1420"
+   LastUpgradeVersion = "1520"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->